pkg/tracker/factory.go

```diff
@@ -3,45 +3,49 @@
 
 package tracker
 
-import "fmt"
+import (
+	"fmt"
+
```

Comment:
```suggestion
```

Reply:
The extra line is put automatically when I save by the linter... It isn't like this for you?